packages/core/src/lib/services/wallet-modules/wallet-modules.service.ts

```diff
@@ -265,6 +298,25 @@ export class WalletModules {
         accounts,
         contractId,
         methodNames,
+        contracts: null,
+      });
+
+      return accounts;
+    };
+
+    wallet.signInMulti = async (params: never) => {
```

Comment:
Rather than having every wallet module be required by _convention_ to implement a `signInMulti()` method that `throw`'s an error if it isn't supported, can we instead _not_ add a `signInMulti()` method in the modules that do not support it, and make **this decorator** check if the method exists and throw the error
```
throw new Error(
        `The signInMulti method is not supported by ${metadata.name}`
      );
```

if it hasn't been implemented?

This has two benefits:
1. If any wallet module doesn't implement the method for whatever reason, callers will always get a consistent error thrown instead of there being a '_signInMulti() is not a function' type of error thrown from our attempt to call it in those cases
2. We don't need to inform wallet module authors about the need to add a stub, ensure their modules have the stub during code reviews, etc.  -- we can treat the lack of implementation as a 'not supported'.

Reply:
Addressed it in this commit: https://github.com/near/wallet-selector/pull/811/commits/8121e5e8f91ffc2ed6a71416a7a51a0db8cf1a95